hoss/spatial.py

```diff
@@ -187,6 +209,71 @@ def get_projected_x_y_index_ranges(
     return x_y_index_ranges
 
 
+def get_x_y_index_ranges_from_coordinates(
+    non_spatial_variable: str,
+    varinfo: VarInfoFromDmr,
+    prefetch_coordinate_datasets: Dataset,
+    coordinates: Set[str],
+    index_ranges: IndexRanges,
+    bounding_box: BBox = None,
+    shape_file_path: str = None,
+) -> IndexRanges:
+    """This function returns a dictionary containing the minimum and maximum
+    index ranges for a pair of lat/lon coordinates, e.g.:
+
+    index_ranges = {'/x': (20, 42), '/y': (31, 53)}
```

Comment:
The initial bit of this description (the bit people are most likely to read) doesn't quite have enough information. I think the disconnect is due to referring to latitude and longitude and then the example having x and y.

Reply:
Also - the names don't actually match what your function produces (currently "projected_x" and "projected_y" for all grids)

Reply:
updated it - https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/802fe0ec04b853a7a7abb4718e2a05b0e8e6ed3c

But may need to revise it  - if I pursue new names to support multiple grids

Reply:
Okay. Thanks for the update. I'll close this comment thread, and we can continue to discuss the multiple-grid things in [this thread](https://github.com/nasa/harmony-opendap-subsetter/pull/15/files#r1785305673). (That's still an issue that will have to be resolved in this PR)